### ISSUE TITLE
Warnings while `npm install`

### ISSUE THREAD

**Issue by starikovs** [angular/universal-starter#527]
I have some warnings while `npm install` but when I did this a few days ago everything worked without any warnings..

```
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN @schematics/angular@0.1.13 requires a peer of @angular-devkit/schematics@0.0.46 but none is installed. You must install peer dependencies yourself.
npm WARN @schematics/schematics@0.0.13 requires a peer of @angular-devkit/schematics@0.0.46 but none is installed. You must install peer dependencies yourself.

```

And when I do `npm ls` it prints:

```
npm ERR! peer dep missing: @angular-devkit/schematics@0.0.46, required by @schematics/angular@0.1.13
```

**Comment by indiefun** [comment]
same error on macOS high sierra (10.13.2)
node --version : v6.12.3
npm --version : 5.6.0

**Comment by CesarDevesa** [comment]
Same issue [https://github.com/angular/angular-cli/issues/8767](url)

**Comment by christianemmanuel** [comment]
How can we fix this? same issue :(

**Comment by wibaldwin** [comment]
same issue here.  Not a great start with Angular...

**Comment by MarkPieszak** [comment]
Is there an issue when running? 
Warnings shouldn't cause you any problems, that's just for authors to be aware of dependencies etc

**Comment by wibaldwin** [comment]
I haven't done much since I am just beginning with Angular but so far going
through the tutorial everything has worked as expected.

**Comment by indiefun** [comment]
update @angular/cli to latest version 1.6.5 will fix this issue

**Comment by CaerusKaru** [comment]
Closing as not an issue with this repo (building from fresh install pulls in CLI ^1.5.0 which resolves to 1.6.8)
